theurgy.lic

```diff
@@ -62,6 +62,69 @@ def clean_altar
   put_in_container($settings.water_holder)
 end
 
+def prayer_bead
```

Comment:
not exactly - you can make the beads with 140+ mech lore (with the method I'm doing it) but it's going to run a check against commune truffanyis which does have a circle 60 requirement.